content/exercises/data-scenarios/vector-needs-reformatting/index.md

```diff
@@ -17,7 +17,7 @@ listing:
 You read a research publication about correlation between the chemistry of blood samples
 taken from 12 kittens found on the Greenland Ice Sheet and their subjective cuteness as
 rated by 100 randomly-selected graduate students. The paper includes graphs and some
-limited data tables, but after searching for hours you can't find the full data file(s)
+limited data tables, but after searching for hours you cannot find the full data file(s)
```

Comment:
`can not` instead of `cannot`? I think that's the most clear.

Reply:
66f73d7

Reply:
https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference

> Both cannot and can not are perfectly fine, but cannot is far more common and is therefore recommended, especially in any kind of formal writing.

This was the reason I went with `cannot` over `can not`. I actually instinctively wrote it as `cannot`,  second-guessed myself, and then looked this up :laughing: